config/config-example.js

```diff
@@ -401,6 +401,15 @@ exports.replsocketmode = 0o600;
  */
 exports.disablehotpatchall = false;
 
+/**
+ * forcedpublicprefixes - user ID prefixes which will be forced to battle publicly.
+ * Battles involving user IDs which begin with one of the prefixes configured here
+ * will be unaffected by various battle privacy commands such as /modjoin, /hideroom
+ * or /ionext.
+ * @type {string[]}
+ */
+exports.forcedpublicprefixes = [];
```

Comment:
I'm accepting `string` prefixes because a) its a `.js` file and you cant `as ID` stuff, and b) the uppercase prefix is prettier for display purposes.

I gave up on making prefixes more finely grained (lots of edge cases and weird interactions) - instead, these only apply to battles and only to a preset amount of commands involving privacy)